_posts/2018-04-05-to-do.md

```diff
@@ -0,0 +1,39 @@
+---
+layout: post
+title: To-Do
+date: 2018-04-05
+categories: projects
+description: A flask web application to keep organized with a list of tasks 
+author: Spencer Mycek
+author-image: https://avatars1.githubusercontent.com/u/32203066?s=460&v=4
+author-bio: First year computer security with an interest in web development 
+author-email: Spence.mycek@gmail.com
+author-social:
+  github: https://github.com/SpencerMycek
+  linkedin: https://www.linkedin.com/in/spencer-mycek-5aa926152/
+---
+
+## To-Do
+The idea for To-Do came to me as a way to organize what I have to do in terms of homework and other important matters. At the time I had not heard of Trello or other similar things, but even after hearing about them I decided to continue with the creation of To-Do.
+I created To-Do with Flask, a mySQL database, and it is currently running on an ubuntu server.
+
+### Flask
+Flask is a web framework. It tools and technology that makes it easier to build a web application. It is also a micro-framework, meaning it has a very small list of dependencies that a user needs to grow and build if they need more plug-ins.
+Writing the flask-side of this application was a fun learning experience. I already knew the python required, but I had to learn the flask to back it up. I also had to tackle each bug and issue as it arose. I had to adapt my code as I learned better practices for flask, such as the application factory model. The application factory model builds each separate module, such as error handling or authentification, is built and connected to the application instance during startup.
```

Comment:
"Writing the Flask side..."

Reply:
`s/python/Python/`
`s/flask/Flask/`
`s/authentification/authentication/`
Try "The application factory model builds and attaches each individual module (such as error handling or authentication) to the application instance during startup."